src/components/Tooltip/Tooltip.tsx

```diff
@@ -79,3 +66,38 @@ export const Tooltip = ({children, content, placement = DEFAULT_PLACEMENT}: Tool
         </>
     );
 };
+
+function useTooltipVisible(anchor: HTMLElement | null, {openDelay, closeDelay}: TooltipDelayProps) {
+    const anchorHovered = useAnchorHovered(anchor);
+    const [tooltipVisible, showTooltip, hideTooltip] = useBoolean(false);
+    useEffect(() => {
+        if (anchorHovered && !tooltipVisible) {
+            const timeout = setTimeout(showTooltip, openDelay);
+            return () => {
+                clearTimeout(timeout);
+            };
+        }
+        if (!anchorHovered && tooltipVisible) {
+            const timeout = setTimeout(hideTooltip, closeDelay);
+            return () => {
+                clearTimeout(timeout);
+            };
```

Comment:
Can we always clearTimeout in hook's cleanup to avoid code duplication? It's no harm to do so.

Reply:
Fixed